studio/pages/projects.tsx

```diff
@@ -2,17 +2,22 @@ import ProjectList from 'components/interfaces/Home/ProjectList'
 import { AccountLayout } from 'components/layouts'
 import OrganizationDropdown from 'components/to-be-cleaned/Dropdown/OrganizationDropdown'
 import Connecting from 'components/ui/Loading/Loading'
-import { useStore } from 'hooks'
+import { useOrganizationsQuery } from 'data/organizations/organizations-query'
+import { useAutoProjectsPrefetch } from 'data/projects/projects-query'
 import { IS_PLATFORM } from 'lib/constants'
-import { observer } from 'mobx-react-lite'
 import { NextPageWithLayout } from 'types'
 
 const ProjectsPage: NextPageWithLayout = () => {
-  const { app } = useStore()
+  const { data: organizations, isLoading: _isLoading } = useOrganizationsQuery()
+  useAutoProjectsPrefetch()
+
+  // We'll continue the loading state if there's no organizations
+  // because we're waiting for the default organization to be created
+  const isLoading = _isLoading || (organizations?.length ?? 0) <= 0
```

Comment:
what if the account just has no organizations though? will it be stuck in a loading state?